app/models/vm_or_template.rb

```diff
@@ -188,6 +188,7 @@ class VmOrTemplate < ApplicationRecord
   scope :archived,     ->       { where(:ems_id => nil, :storage_id => nil) }
   scope :orphaned,     ->       { where(:ems_id => nil).where.not(:storage_id => nil) }
   scope :retired,      ->       { where(:retired => true) }
+  scope :not_active,   ->       { where(:ems_id => nil) }
```

Comment:
Archived means something more specific for VMs than just "not connected to an EMS"